system/service.py

```diff
@@ -941,6 +941,7 @@ class FreeBsdService(Service):
 
     platform = 'FreeBSD'
     distribution = None
+    sysrc_cmd = "/usr/sbin/sysrc"
```

Comment:
I would use the module's find_bin function to locate sysrc as you cannot always trust it to be in the default location

Another point is that this is new in v9 (iirc) so this will break the module for those using v8 and below. I know they are ancient and unsupported ... but people still run these systems. So i would fallback to the current system if sysrc is not available.

And this is just an informative note, I'm trying to deprecate this module in favor of 'per system' ones, so I might end up pulling these changes into a bsdrc module. But systemctl is only one I've had time to separate.